test/integration/stepper/Stepper.spec.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-env mocha */
+import React from 'react';
+import {shallow} from 'enzyme';
+import {assert} from 'chai';
+import Step from 'src/Stepper/Step';
+import StepConnector from 'src/Stepper/StepConnector';
+import Stepper from 'src/Stepper/Stepper';
+import FontIcon from 'src/FontIcon';
+import getMuiTheme from 'src/styles/getMuiTheme';
+
+describe('<Stepper />', () => {
```

Comment:
We already have a test under `src/Stepper/Stepper.spec.js`. That would be better to move those tests their.